src/main.py

```diff
@@ -1,7 +1,15 @@
+import os
+
 from follower import Follower
 from leader import Leader
 from tagrec import TagRecognition
 
+"""
+Checks if a path exists to /dev/input/js0 and returns a boolean value.
```

Comment:
Why does this path matter to someone who doesn't know the system?
`"Returns whether or not a Bluetooth joystick is connected."`